src/Authentication/OldStyleTokens/AccessTokenUrlAuthenticator.php

```diff
@@ -25,7 +25,8 @@ public function addAuthenticationTokensToApplicationUrl( string $url ): string {
 	/**
 	 * @param class-string $urlGeneratorClass
 	 * @param string[] $requestedParameters
-	 * @return array<string,string>
+	 *
+	 * @return array<string, bool|int|string>
```

Comment:
```suggestion
	 * @return array<string, int|string>
```
`getUpdateToken` and `getAccessToken` should not return booleans